charm/kafka/reactive/kafka.py

```diff
@@ -85,13 +100,11 @@ def get_snap_file_from_charm():
 def waiting_for_zookeeper():
     hookenv.status_set('blocked', 'waiting for relation to zookeeper')
 
-
```

Comment:
The extra newline should stay. It's a PEP8 thing. We should add PEP8 linting to our Makefiles...